openapi-cli/src/test/resources/ballerina-to-openapi/rb_scenario11.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/http;
+
+listener http:Listener helloEp = new (9090);
+type TagType record {
+    int id;
+    string[] category;
+};
```

Comment:
Shall we add a line between each record and remove the extra line at the end of the file?